ddr/ddr/Models/GameEntities/Battery.swift

```diff
@@ -0,0 +1,83 @@
+
+import UIKit
+import SpriteKit
+
+class Battery: Oncomer {
+  
+  static let IMPACT_FILE = Bundle.main.path(forResource: "charge.mp3", ofType: nil)!
+  static let FLAP_VELOCITY_CONVERSION: CGFloat = 1
+  static let DEFAULT_SPEED: CGFloat = 1
+  static let DEFAULT_Y: CGFloat = GameScene.HEIGHT * 3 / 4
+
+  let impact: Emitter
+  
+  // factory method
+  static func getTrainingBattery(position: ScreenThird, speed: CGFloat = Battery.DEFAULT_SPEED) -> Battery{
+    let battery = Battery(position: position, speed: speed)
+    battery.collisionEffects = [] // training bats can't actually hurt you
```

Comment:
please remove references to bats, and change any logic that does not make sense in the context of a battery (because this function does not make sense as it is written now, at least to me)